### PR TITLE
Cbcolleg translate sentences to commands!

### PR DESCRIPTION
**Pull request by CBColleg3**

### CODE REVIEW

**Review by mwang840 — COMMENTED** [review]
The === looks good

File: src/Classes/chatgptrequest.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+
+
+import Phaser from 'phaser';
+import axios, { AxiosError } from 'axios';
+import chatGPTKey from '../secrets/secretkeys';
+
+import { OpenAIApi, Configuration, ChatCompletionResponseMessageRoleEnum } from "openai";
```
Comment (mwang840) [inline src/Classes/chatgptrequest.ts:7]: Does this grabs the chatgpt api and is the open ai for the enemy player?

**Review by Izahe — APPROVED** [review]
cool

**Review by mwang840 — APPROVED** [review]
Not too bad, saw the demo on your localhost and the player jumps respectfully. Good work!

File: src/Classes/chatgptrequest.ts (hunk 1 above)
Comment (CBColleg3) [reply src/Classes/chatgptrequest.ts:7]: No this is for both players, and yes it grabs te chatgpt ai.

**Pull request merged**